ldms/src/ldmsd/ldmsd.h

```diff
@@ -1224,7 +1224,7 @@ typedef struct ldmsd_auth {
 
 /* Key (name) of the default auth -- intentionally including SPACE as it is not
  * allowed in user-defined names */
-#define DEFAULT_AUTH " _DEFAULT_AUTH_ "
+#define DEFAULT_AUTH "auth_default"
```

Comment:
Is a configuration object created for "auth_default?" IOW, if a user does this:

```
prdcr_add name=X host=node-1 ... auth="auth_default"
```

It will use the -a, -A provided on the command line?

Reply:
Yes.

The default auth config object was already there, with a name that cannot be specified by users because it contains spaces. The reason was probably to avoid name collision.

Reply:
Thanks for the explanation @narategithub